library/common/1.0.0/templates/lib/service/serviceTypeConfig/_sessionAffinity.tpl

```diff
@@ -0,0 +1,42 @@
+{{/* Service - Session Affinity */}}
```

Comment:
I believe we don't need this as we are removing load balancer services ?

Reply:
It technically can be used with other types too.